dockci/notifications.py

```diff
@@ -0,0 +1,31 @@
+""" Module for notification integration, for DockCI """
+import hypchat
+
+# pylint:disable=too-few-public-methods
+
+
+class HipChat():
+    """ HipChat Class """
+
+    def __init__(self, apitoken, room):
+        self.token = apitoken
+        self.roomname = room
+        self.roomid = 0
```

Comment:
if you always set this later on, set the initial value to None (0 _could_ be mistaken for a legitimate ID)